tests/test_user_login.py

```diff
@@ -0,0 +1,65 @@
+"""
+This file holds user login test cases.
+"""
+import unittest
+import json
+from api import create_app
+
+
+class AuthTestCase(unittest.TestCase):
+    """
+        Test case for User login.
+    """
+
+    def setUp(self):
+        """Set up test variables."""
+        self.app = create_app(config_name="testing")
```

Comment:
I am curious about what is happening on this line. Please colour on what this line of code does.

Reply:
It passes in the testing configuration